src/main/java/uk/gov/ons/census/casesvc/service/FulfilmentRequestService.java

```diff
@@ -29,7 +29,11 @@
               "P_OR_I4", // HOUSEHOLD_INDIVIDUAL_RESPONSE_REQUEST_NI_PRINT,
               "P_UAC_UACIP1", // INDIVIDUAL_UAC_ENGLAND_PRINT_LETTER
               "P_UAC_UACIP2B", // INDIVIDUAL_UAC_WALES_BILINGUAL_PRINT_LETTER
-              "P_UAC_UACIP4" // INDIVIDUAL_UAC_NI_PRINT_LETTER
+              "P_UAC_UACIP4", // INDIVIDUAL_UAC_NI_PRINT_LETTER
+              "P_LP_ILP1", // INDIVIDUAL_QUESTIONNAIRE_LARGE_PRINT_ENGLAND_LETTER
+              "P_LP_ILP2", // INDIVIDUAL_QUESTIONNAIRE_LARGE_PRINT_WALES_ENGLISH_LETTER
+              "P_LP_ILP2W", // INDIVIDUAL_QUESTIONNAIRE_LARGE_PRINT_WALES_WELSH_LETTER
+              "P_LP_IL4" // INDIVIDUAL_QUESTIONNAIRE_LARGE_PRINT_NI_LETTER
```

Comment:
Is there any need to add these codes in here? As far as I can tell the only reason for this list is for codes that require a HI case, if we simply leave it out here then we don't get the unwanted HI case.

Reply:
Just checking you are aware that these codes require a HI case if case_type is originally HH (https://collaborate2.ons.gov.uk/confluence/pages/viewpage.action?spaceKey=SDC&title=RM+Product+Library)

Reply:
They shouldn't require a HI case, since we're not including a UAC there is no reason for a HI case. I believe that document is incorrect as it was being written under the assumption we'd have to do extra work to _avoid_ generating the HI case.